[OSDOCS#18857]: Added a new module for Fencing Validator script (#110148)
openshift/openshift-docs: modules/Installation-fencing-validator-prerequisites.adoc

:_mod-docs-content-type: CONCEPT
[id="installation-fencing-validator-prerequisites_{context}"]
= Fencing validator script prerequisites

[role="_abstract"]

> Use the `fencing_validator` script to verify your fencing configuration on a two-node {product-title} cluster. This script, deployed automatically by the Machine Config Operator, ensures that power management interfaces are correctly configured to prevent data corruption during a node failure. To run it, ensure the jq utility is installed, and you have both Kubernetes API access (oc) and SSH access to the control-plane nodes.

Reviewer: ```suggestion
Use the `fencing_validator` script to verify your fencing configuration on a two-node {product-title} cluster. This script, deployed automatically by the Machine Config Operator, ensures that power management interfaces are correctly configured to prevent data corruption during a node failure. To run it, ensure the `jq` utility is installed, and you have both Kubernetes API access (`oc`) and SSH access to the control-plane nodes.
```

:_mod-docs-content-type: CONCEPT
[id="installation-fencing-validator-prerequisites_{context}"]
= Fencing validator script prerequisites

[role="_abstract"]

> You need certain prerequisites and dependencies to successfully run the `fencing_validator` script. 

Reviewer 2: ```suggestion
To run the `fencing_validator` script on a Two-Node with Fencing {product-title} cluster, you need `jq` on the node, Kubernetes API access through `oc` or `oc debug`, and SSH access to both control-plane nodes when you use SSH transport. The script runs only on this cluster topology, where the Machine Config Operator deploys it.```

Reviewer 2: I used Steven's abstract  command and here is what it suggested. You need more than what you have because you don't have a clear what and why.

> * The light-weight command-line JSON processor `jq` must be available on the node. By default, Red Hat Enterprise Linux CoreOS(RHCOS) includes the `jq` utility.

Reviewer 2: ```suggestion
* The lightweight command-line JSON processor `jq` must be available on the node. By default, Red Hat Enterprise Linux CoreOS (RHCOS) includes the `jq` utility.
```
